Jenkinsfile

```diff
@@ -9,6 +9,9 @@
 *
 */
 
+@Library('shared-pipelines') import org.zowe.pipelines.nodejs.NodeJSPipeline
+def lib = library("jenkins-library").org.zowe.jenkins_shared_library
+
```

Comment:
Do we need to import the shared libraries here?

Reply:
No, I forgot to remove them.

So they are removed and also @crawr's solution is added in the Smoke Test step.